src/Raven.Server/Documents/Patch/V8/ObjectBinderEx.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using V8.Net;
+
+namespace Raven.Server.Documents.Patch.V8
+{
+    public class ObjectBinderEx<T> : ObjectBinder where T : class
+    {
+        public ObjectBinderEx() : base()
+        {
+        }
+
+        public T ObjCLR
```

Comment:
What is this name?

Reply:
this is the typed bound object (the original binder's property Object is of generic type 'object')
it is convenient to use in child classes 
renamed it to ObjClr